Lib/axis.py

```diff
@@ -2192,6 +2192,8 @@ def __setitem__(self, index, value):
 
     def __setslice__(self, low, high, value):
         # Hack to prevent netCDF overflow error on 64-bit architectures
+        if(self.isUnlimited() and self.isTime() and (high >= Max32int)):
```

Comment:
shouldn't it raise an exception if you do time[121:122] ? Since time does not exists. What happen to the data? The get `undefined` value? Or they copy the value? sounds weird.